src/INA138.cpp

```diff
@@ -0,0 +1,22 @@
+#include "INA138.hpp"
+
+namespace LVSS {
+
+INA138::INA138(IO::ADC& adc0) : ADC(adc0) {
+}
+
+uint32_t INA138::readCurrent(IO::ADC& adc0) {
+    //Gets voltage from adc
+    uint32_t voltage = static_cast<uint32_t>(adc0.read() * 1000);
+    voltage *= fixedPoint;
```

Comment:
Don't need to multiply the input by anything because the raw value is already an integer
```suggestion
```